defenseunicorns/uds-core
# feat: uds core docs

## Description
Integrated docs between `uds` and other repos. `uds-core` is the initial repo to be integrated.

The module repo's need to add a go.mod file to their `docs` directory which is then pulled in by the `uds` repo to add the docs to the hugo site. 

The module repo's need to pr changes before the `uds` repo's pr can be pushed in due to failing CI runs because the main branch of the module repo's doesn't have the go.mod file present.

I attempted copy and paste the doc files from `uds` with minimal changes, only changes would be links needing to reference urls instead of static local files. 

Hopefully the only real changes are the:
* go.mod file
* frontmatter addition to doc files
* syntax changes for links and alert messages in docs

#
Will let @Jessy-Morris take the first crack at reviewing this since this is her domain!
#

# How to review
1.  checkout the [core-docs-import](https://github.com/defenseunicorns/uds/tree/core-docs-import) branch on `uds` 
2.  run `npm run start` 
3.  open `localhost:1313` in the browser and there should be `UDS Core` tab to view the docs in this uds-core branch. 

## Type of change


## Checklist before merging
